### PR TITLE
RealPython home page displayed on no user search.

### LINKED ISSUE #1001: Modify `.realpython` blank command to embed a url to the site
## Description

When no input is used in the command, the embed sent should have the url to the website. 

## Reasoning

Currently a blank `.realpython` command raises an error, but it could be helpful to have a link to the Real Python homepage instead.

## Proposed Implementation

I have an idea for how this could work, but this can be a good first issue for someone starting out. 

## Would you like to implement this yourself?

- [ ] I'd like to implement this feature myself
- [X] Anyone can implement this feature

### PR DESCRIPTION
**Pull request by Xithrius**
## Relevant Issues

Closes #1001

## Description

Displays [RealPython's Home Page](https://realpython.com/) when a user provides no query during a `.realpython` call.

Example:
image

## Did you:

- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] Read all the comments in this template?
- [x] Ensure there is an issue open, or link relevant discord discussions?
- [x] Read and agree to the [contributing guidelines](https://pythondiscord.com/pages/contributing/contributing-guidelines/)?

### CODE REVIEW

**Review by brad90four — APPROVED** [review]
This looks like another top notch PR from our very own triage lead. Thanks @Xithrius

File: bot/exts/utilities/realpython.py [hunk 1]
```diff
@@ -32,13 +31,21 @@ def __init__(self, bot: Bot):
 
     @commands.command(aliases=["rp"])
     @commands.cooldown(1, 10, commands.cooldowns.BucketType.user)
-    async def realpython(self, ctx: commands.Context, amount: Optional[int] = 5, *, user_search: str) -> None:
+    async def realpython(self, ctx: commands.Context, amount: Optional[int] = 5, *, user_search: str = None) -> None:
```
Comment (LukeHankey) [inline bot/exts/utilities/realpython.py:34]: If defaulting to None, would this not be an Optional[str]?

```suggestion
    async def realpython(self, ctx: commands.Context, amount: Optional[int] = 5, *, user_search: Optional[str] = None) -> None:
```

Comment (Xithrius) [reply bot/exts/utilities/realpython.py:34]: This is invalid syntax.

Comment (Xithrius) [reply bot/exts/utilities/realpython.py:34]:
```
File "/home/xithrius/Repositories/PythonDiscord/sir-lancebot/bot/exts/utilities/realpython.py", line 34
    async def realpython(self, ctx: commands.Context, amount: Optional[int] = 5, *, Optional[user_search]: str = None) -> None:
                                                                                            ^
SyntaxError: invalid syntax
```

Comment (Xithrius) [reply bot/exts/utilities/realpython.py]: Fixed in dc4bac651f71707b1e2144ac3b94e8ea94b86c7e.

**Review by ChrisLovering — APPROVED** [review]

**Pull request merged**